packages/core/src/snapshot.js

```diff
@@ -118,6 +118,7 @@ function getSnapshotOptions(options, { config, meta }) {
       disableCache: config.discovery.disableCache,
       captureMockedServiceWorker: config.discovery.captureMockedServiceWorker,
       captureSrcset: config.discovery.captureSrcset,
+      captureResponsiveAssets: config.discovery.captureResponsiveAssets,
```

Comment:
This should not be needed, we should call this explicitly again if there is `enableJS` true. If we want to keep it then we need confirmation from product.

Reply:
Ideally we should do this automatically.